Makefile

```
@@ -119,6 +119,16 @@ down-local:
 	$(STANDALONE_DIRECTORY)/docker-compose-core.yaml -f $(STANDALONE_DIRECTORY)/docker-compose-local.yaml \
 	down -v
 
+run-dependencies:
+	@LOCAL_GOOS=$(LOCAL_GOOS) LOCAL_GOARCH=$(LOCAL_GOARCH) docker-compose -f \
+	$(STANDALONE_DIRECTORY)/docker-compose-core.yaml \
+	up --build -d
```

No need to pass `LOCAL_*` variables as well as `--build` flag.
They are used by `docker-compose-local.yaml` to build docker images with local changes.

```suggestion
	@docker-compose -f $(STANDALONE_DIRECTORY)/docker-compose-core.yaml up -d
```